test/evergreen.yml

```diff
@@ -2623,7 +2623,7 @@ tasks:
           script: |
             set -o errexit
             set -o verbose
-            ${test_env_vars|} ../../../tools/run_parallel.sh 'nice ../../../test/checkpoint/recovery-test.sh "${data_validation_stress_test_args} -s 6" WT_TEST.$t' 120
+            ${test_env_vars|} ../../../tools/run_parallel.sh 'nice ../../../test/checkpoint/recovery-test.sh "${data_validation_stress_test_args} -s 6" WT_TEST.$t' 100
```

Comment:
@jeremythorp , I actually wonder why the test still preserves the directory of previous successful iterations? Usually test/format stress testing removes its previous run.

Reply:
The tests run in each iteration overwrite the directories of the previous iteration, and that's why (once the `nohup.out.*`  files are sorted out) the disk usage remains stable.